test_runner/batch_others/test_branch_behind.py

```diff
@@ -1,6 +1,9 @@
 import subprocess
 from fixtures.zenith_fixtures import PostgresFactory, ZenithPageserver
 
+import logging
+import fixtures.log_helper  # configures loggers
```

Comment:
There are no fixtures in `log_helper`, so I doubt that it really belongs to `fixtures`.

Reply:
Couldn't find a better place for log_helper, any suggestions? Even though there is no real pytest fixture in log_helper, I think it has purpose similar to utils and zenith_fixtures in the same directory.

Reply:
I can propose rename `fixtures` to `lib` or `utils`, as there is already utils.py which is used outside fixtures; or do something like
```
test_runner/
├── fixtures/
│   ├── zenith_fixtures.py
├── lib/
│   ├── utils.py
│   └── log_helper.py
```
but it seems like more hassle than needed.